modules/RTC/TraceablePeerConnection.js

```diff
@@ -1043,6 +1052,18 @@ function extractSSRCMap(desc) {
     return ssrcMap;
 }
 
+/**
+ * @param allSsrcs all the send ssrcs for this client.
```

Comment:
'allSsrcs' does not match argument name 'isSelected'

Reply:
fixed